### PR TITLE
SEO and Accessibility changes

### PR DESCRIPTION
**Pull request by jindaliiita**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fixes

Test URLs:
- Original: https://www.sunstar.com/healthy-thinking/4667
- Before: https://main--sunstar--hlxsites.hlx.live/healthy-thinking/4667
- After: https://acc-seo-fix--sunstar--hlxsites.hlx.live/healthy-thinking/4667

- [ ]  **If you are adding a new block or a variation to an existing block**, please fill below:

  Block library path: https://<branch>--sunstar--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/<your-block>&index=0

### CODE REVIEW

**Comment by jindaliiita** [comment]
@ehrro We have made changes related to accessibility. Could you please review ?
There is an issue with the color combination in the footer.

**Review by bosschaert — APPROVED** [review]

File: blocks/social/social.js [hunk 1]
```diff
@@ -23,9 +23,11 @@ export default async function decorate(block) {
     const a = x.querySelector('a');
```
Comment (hparwani2) [inline blocks/social/social.js:23]: what is x? The variable name should describe element.

**Comment by jindaliiita** [comment]
@ehrro Can you please also check this

**Comment by sdmcraft** [comment]
I see a net accessibility score increase of 5 points.
From
image

To
image

The desktop scores still are on the lower side and something to look (maybe separately from this PR)

**Pull request merged**